libs/media-player/src/audio-clip-player.cy.tsx

```diff
@@ -0,0 +1,44 @@
+import { AudioClipPlayer } from './lib/audio-clip-player';
+import { DefaultPlayPauseButton } from './lib/default-play-pause-button';
+
+describe('<AudioClipPlayer />', () => {
```

Comment:
You need to stub the Audio api in Cypress [as in this comment](https://github.com/cypress-io/cypress/issues/1750#issuecomment-392132279).

Reply:
We did this.